### PR TITLE
feat(rebalance): introduce rebalance rpc timeout

### PR DESCRIPTION
**Pull request by savonarola**
Fixes [EMQX-13239](https://emqx.atlassian.net/issues/EMQX-13239)

Release version: e5.9 

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [x] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [x] Schema changes are backward compatible

[EMQX-13239]: https://emqx.atlassian.net/browse/EMQX-13239?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/emqx_node_rebalance/src/emqx_node_rebalance_api.erl [hunk 1]
```diff
@@ -614,6 +630,14 @@ fields(local_status_enabled) ->
                     required => false
                 }
             )},
+        {connection_eviction_rpc_timeout,
+            mk(
+                emqx_schema:timeout_duration_s(),
```
Comment (thalesmg) [inline apps/emqx_node_rebalance/src/emqx_node_rebalance_api.erl:635]: Here, we have `timeout_duration_s`, but in `rebalance_start` it is `timeout_duration_ms`.  Is it intentional?

Comment (savonarola) [reply apps/emqx_node_rebalance/src/emqx_node_rebalance_api.erl]: In short, yes, but now I decided to simplify and get ms everywhere.

File: apps/emqx_node_rebalance/src/emqx_node_rebalance.erl [hunk 2]
```diff
@@ -377,16 +415,22 @@ get_stats(_State, Data) -> Data.
...
-    case apply(emqx_node_rebalance_proto_v3, F, [Nodes | A]) of
+wrap_multicall(MulticallResult, Nodes) ->
+    case MulticallResult of
         {Results, []} ->
             case lists:partition(fun is_ok/1, lists:zip(Nodes, Results)) of
                 {OkResults, []} ->
                     [{Node, ok_result(Result)} || {Node, Result} <- OkResults];
                 {_, BadResults} ->
+                    ?tp(warning, emqx_node_rebalance_bad_nodes, #{
+                        bad_nodes => BadResults
```
[the remark this answers is not in the document]
Comment (savonarola) [reply apps/emqx_node_rebalance/src/emqx_node_rebalance.erl]: ✅

**Review by thalesmg — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-6647)

**Pull request merged**